### ISSUE TITLE
Add platform sub pages

### ISSUE THREAD

**Issue by karatechops** [hpe-dev-incubator/hpe-dev-portal#26]
![Screen Shot 2019-10-25 at 10 50 25 AM](https://user-images.githubusercontent.com/5983843/67581035-4fff0e00-f715-11e9-8cde-971bcf51f9ac.png)

**Comment by karatechops** [comment]
Example architecture:
```
content
\_ hpe-simplivity
  \_ home.md
    | architecture-and-object-model.md   
```
`home.md` example
```
---
title: Simplivity
version: v 6.01.8964
description: A hyperconverged platform uniting best-in-class data services with the world's bestselling server.
image: 
frontpage: true
priority: 2
tags: ["Simplivity"]
# Aside should handle strings for section headings and md references
aside: [{ title: "Explore the API" }, "architecture-and-object-model.md", "my-other-thing.md"]
---

## HPE OmniStack REST API

The HPE OmniStack REST API enables tool developers and integrators to manage HPE OmniStack assets efficiently, intuitively, and securely. Anyone can use the REST API to gather information about, analyze, configure, and troubleshoot HPE OmniStack hosts and federations. Moreover, tools developers can use the REST API to automate these tasks.
```

`aside` page query data:
```
"aside": [
  { title: 'Exploring the API' },
  { title: 'Architecture and object model', path: 'architecture-and-object-model' },
  { title: 'my page title', path: 'my-page-path' }
]
```

**Comment by IanKBovard** [comment]
closed by #30
